src/main/java/bio/terra/workspace/service/resource/controlled/ControlledBigQueryDatasetAttributes.java

```diff
@@ -0,0 +1,25 @@
+package bio.terra.workspace.service.resource.controlled;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class ControlledBigQueryDatasetAttributes {
+  private final String projectId;
+  private final String datasetName;
+
+  @JsonCreator
+  public ControlledBigQueryDatasetAttributes(
+      @JsonProperty("projectId") String projectId,
+      @JsonProperty("datasetName") String datasetName) {
+    this.projectId = projectId;
+    this.datasetName = datasetName;
+  }
+
+  public String getProjectId() {
+    return projectId;
```

Comment:
This was mostly important for converting to API objects. We definitely need project ID there since it's part of the unique identifier for a dataset, but I realized that's the only place it's really convenient to have it as part of the resource. 
Refactored this out so projectId is passed in during conversion to API object instead of stored as an attribute.